### PR TITLE
Fix tests and add CI

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,11 +1,11 @@
 [tox]
-envlist = py{27,34}-integration, syntax
+envlist = py{27,35}-integration, syntax
 
 [testenv]
 deps =
   pytest
   pytest-cov
-  https://downloads.mesosphere.io/dcos-cli/dcos-0.1.0-py2.py3-none-any.whl
+  https://pypi.python.org/packages/2.7/d/dcos/dcos-0.1.15-py2.py3-none-any.whl#md5=869aeccee18c536ed9335984d851a0ef
```
Comment (jsancio) [inline tox.ini]: We can change this to `dcos==0.1.15`

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,28 @@
-sphinx
-tox
-wheel
--e git+git@github.com:mesosphere/dcos-cli#egg=dcos
+alabaster==0.7.6
+Babel==2.1.1
+-e git+https://github.com/mesosphere/dcos-cli.git@372c3a5811ac537a104a5bf93bfa3fb32df44473#egg=dcos-0.2.0
```
Comment (jsancio) [inline requirements.txt]: Since we now have the dcoscli in PyPI, we can just change this to `dcoscli==0.2.0``

File: requirements.txt [hunk 3]
```diff
@@ -1,4 +1,28 @@
...
+requests==2.8.1
+six==1.10.0
+smmap==0.9.0
+snowballstemmer==1.2.0
+Sphinx==1.3.3
+sphinx-rtd-theme==0.1.9
+toml==0.9.1
+tox==2.2.1
+virtualenv==13.1.2
+wheel==0.24.0
```
Comment (jsancio) [inline requirements.txt:28]: Can you verify that we need all of these dependencies for `dcos-helloworld`? They look a lot like the dependencies for `dcoscli`. Can you just depend on it or `dcos` and let `pip` resolve them?

**Comment by jsancio** [comment]
LGTM. Let's rebase and squash it into one commit. It would be nice to use our TC instead of Travis but okay. Thanks!

File: requirements.txt (hunk 3 above)
Comment (sttts) [inline requirements.txt:28]: It's the output of pip freeze. This is transitive of course.

File: tox.ini (hunk 1 above)
Comment (sttts) [inline tox.ini]: Not dcos=0.2.0?

File: requirements.txt (hunk 2 above)
Comment (sttts) [inline requirements.txt]: done

File: tox.ini (hunk 1 above)
Comment (sttts) [inline tox.ini]: done

**Comment by sttts** [comment]
The travis CI is not for us, but for 3rd party developers. They cannot access our Jenkins.

**Comment by jsancio** [comment]
Thanks! Ship it.

**Pull request merged**